recipes/seroba/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# We don't have a compiler on Conda for C++14, so downloading pre-compiled binary and moving to bin.
```

Comment:
Is it possible for you to update the `kmc` recipe to version `3.0.0` instead of including it here? The existing `kmc` package also provides `kmc`, `kmc_dump`, as well as `kmc_tools`.

Reply:
I'll give it a shot myself, no worries.